x-pack/plugins/ml/public/application/components/anomaly_results_view_selector/anomaly_results_view_selector.tsx

```diff
@@ -17,24 +17,34 @@ import { ML_PAGES } from '../../../../common/constants/locator';
 
 interface Props {
   viewId: typeof ML_PAGES.SINGLE_METRIC_VIEWER | typeof ML_PAGES.ANOMALY_EXPLORER;
+  isSingleMetricViewerDisabled?: boolean;
 }
 
-// Component for rendering a set of buttons for switching between the Anomaly Detection results views.
-
-export const AnomalyResultsViewSelector: FC<Props> = ({ viewId }) => {
+/**
+ * Component for rendering a set of buttons for switching between the Anomaly Detection results views.
+ */
+export const AnomalyResultsViewSelector: FC<Props> = ({
+  viewId,
+  isSingleMetricViewerDisabled = false,
+}) => {
   const locator = useMlLocator()!;
   const navigateToPath = useNavigateToPath();
 
-  const toggleButtonsIcons = useMemo(
+  const toggleButtonsIcons = useMemo<EuiButtonGroupProps['options']>(
     () => [
       {
         id: 'timeseriesexplorer',
-        label: i18n.translate('xpack.ml.anomalyResultsViewSelector.singleMetricViewerLabel', {
-          defaultMessage: 'View results in the Single Metric Viewer',
-        }),
+        label: isSingleMetricViewerDisabled
+          ? i18n.translate('xpack.ml.anomalyResultsViewSelector.singleMetricViewerDisabledLabel', {
+              defaultMessage: 'Selected jobs are not viewable in the Single Metric Viewer',
```

Comment:
Can you easily tell if only 1 job is selected? If so, could conditionally use text of `Selected job is not visible...`

Reply:
Added in https://github.com/elastic/kibana/pull/134048/commits/2e5df827b5eb42065d421f38f2641c9dc9ed506d